frontend/src/components/Navigationbar.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  NavbarText
+} from 'reactstrap';
+
+const Navigationbar = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => setIsOpen(!isOpen);
```

Comment:
Lets try to keep these to either functional or class components. [Here](https://imgur.com/CO0B4U6) is an example of how to do that.